entries/t/thalia.de.json

```diff
@@ -12,31 +11,10 @@
     ],
     "regions": [
       "de",
-      "bg",
-      "be",
-      "ee",
-      "fi",
-      "fr",
-      "gr",
       "at",
       "ch",
-      "nl",
-      "lu",
-      "se",
-      "dk",
-      "it",
-      "gb",
-      "no",
-      "ie",
-      "pt",
-      "hr",
-      "cz",
-      "lt",
-      "ro",
-      "lv",
-      "mt",
-      "si",
-      "is"
+      "li",
+      "lu"
```

Comment:
Why just these 4 countries? https://www.thalia.de/shop/hilfe-versand/show/#D2

Reply:
While the site ships to most EU countries, the site is only available in German which would mean that's their primary market. That was my reasoning at least